lang/julia/Portfile

```diff
@@ -41,8 +41,8 @@ if {[option gpg_verify.use_gpg_verification]} {
     checksums-append \
                     ${name}-${version}-full${extract.suffix}.asc \
                     size    866 \
-                    rmd160  aed07c2e553aec2a030491a2a9c69206b69292ad \
-                    sha256  3128c1ae0392008c92798b4e4cbd076088579fcbfbb11a2fdeb5a47a136c3857
+                    rmd160  08b5b3641f3678e508bacf9b7074abdaa37eb0e1 \
+                    sha256  26ae4fc9ea03b9ba6d9ea126d4671f6f0e1af11bc8ec23f9fd29e75c61de2906
```

Comment:
Doesn't the size checksum also need updating here ?

Reply:
> Doesn't the size checksum also need updating here ?

No—this is a gnupg `.asc` signature of fixed length.

In fact, I recommend removing the `rmd160` and `sha256` hashes because they're unnecessary. There's no reason to hash check a cryptological signature.

Reply:
I agree with @essandess.